cmd/recommend.go

```diff
@@ -16,6 +18,16 @@ var recommendCmd = &cobra.Command{
 	Short: "Recommend Policies",
 	Long:  `Recommend policies based on container image, k8s manifest or the actual runtime env`,
 	RunE: func(cmd *cobra.Command, args []string) error {
+		//Condition to check if at least one Container image name is passes as an argument
+		if len(recommendOptions.Images) < 1 {
+			fmt.Printf("Error: At least one Container image is required as an argument.\n")
+			/*
+				if err := cmd.Help(); err != nil {
+					return err
+				}
+			*/
+			return nil
```

Comment:
```suggestion
		if len(recommendOptions.Images) < 1 {
			return errors.New("at least one container image is required as an argument.")
```

This way we preserve an exitcode which signifies an error, keeping it POSIX compliant.